templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,8 @@
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+<%- dirnames = get_directories.map {|d| d.path } -%>
+<%- filenames = get_files.map {|f| f.path} -%>
+<%- configfilenames = get_configfiles.map {|cf| cf.path} -%>
+<%- (dirnames + filenames + configfilenames).each do |name| -%>
+Copy-Item (Join-Path $toolsDir '<%= name %>') <%= platform.convert_to_windows_path(name) %> -Force
```

Comment:
@ferventcoder rather than copy over the directories recursively, I decided it would be a better approach to copy over all the files explicitly, since we have that information. This *should* include empty directories.